package.json

```diff
@@ -54,6 +54,6 @@
 		"pa11y": "./bin/pa11y"
 	},
 	"scripts": {
-		"test": "make ci"
+		"test": "snyk test && make ci"
```

Comment:
How does the `snyk` command get installed? Maybe it should be in dev dependencies in `package.json`? Otherwise people have to remember to install it themselves.

Also `npm test` is never run in CI – so it shouldn't go here. It should be in [`.travis.yml`](https://github.com/pa11y/pa11y/blob/master/.travis.yml#L27) I think :)